pkg/channel/message_dispatcher.go

```diff
@@ -263,12 +264,16 @@ func (d *MessageDispatcherImpl) sanitizeURL(u *url.URL) *url.URL {
 }
 
 // dispatchExecutionTransformer returns Transformers based on the specified DispatchExecutionInfo
```

Comment:
here too? 
```
based on the specified destination and DispatchExecutionInfo
```

Reply:
good call - will fix this and above comment!